plugins/MauticCrmBundle/Command/PushActivitiesToPipedriveCommand.php

```diff
@@ -0,0 +1,138 @@
+use MauticPlugin\MauticCrmBundle\Integration\Pipedrive\Export\LeadExport;
+use MauticPlugin\MauticCrmBundle\Integration\PipedriveIntegration;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class PushActivitiesToPipedriveCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('mautic:integration:pipedrive:pushactivities')
+            ->setDescription('Pushes the data from Mautic to Pipedrive')
+            ->addOption(
+                'time-interval',
+                't',
+                InputOption::VALUE_OPTIONAL,
+                'Send time interval to check updates, it should be a correct php formatted time interval in the past eg:(15 minutes)',
+                '15 minutes'
+            );
+
+        parent::configure();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $integrationHelper = $this->getContainer()->get('mautic.helper.integration');
+        /** @var PipedriveIntegration $integrationObject */
+        $integrationObject = $integrationHelper->getIntegrationObject(PipedriveIntegration::INTEGRATION_NAME);
+        $em                = $this->getContainer()->get('doctrine')->getManager();
+
+        if (!$integrationObject->getIntegrationSettings()->getIsPublished()) {
+            $output->writeln('Pipedrive integration id disabled.');
+
+            return;
+        }
+
+        $output->writeln('<comment>Push activities</comment>');
+
+        /** @var ActivitiesPipedriveExport $activitiesExport */
+        $activitiesExport = $this->getContainer()->get('mautic_integration.pipedrive.export.activities');
+        $activitiesExport->setIntegration($integrationObject);
+
+        $pushed = 0;
+        $start  = 0;
+        $limit  = 50;
+
+        // now() - time-interval
+        $timeInterval     = str_replace('--', '-', '-'.$input->getOption('time-interval'));
+        $lastPossibleSync = (new \DateTime())->setTimestamp(strtotime($timeInterval));
+
+        $progress = ProgressBarHelper::init($output, $limit);
+
+        while (true) {
+            $integrationEntities = $this->getIntegrationEntities($integrationObject, $start, $limit);
+
+            if (!$integrationEntities) {
+                break;
+            }
+
+            /** @var IntegrationEntity $integrationEntity */
+            foreach ($integrationEntities as $integrationEntity) {
+                $internal         = $integrationEntity->getInternal();
+                $lastActivitySync = null;
+                if (isset($internal['last_activity_sync'])) {
+                    $lastActivitySync = (new \DateTime())->setTimestamp($internal['last_activity_sync']);
```

Comment:
Is it any different than this?
```suggestion
                    $lastActivitySync = new \DateTimeImmutable($internal['last_activity_sync']);
```